application/src/messenger.cpp

```diff
@@ -69,6 +70,12 @@ void Messenger::requestMessageList(const QString &recipientId) const
     m_requester->sendRequest(Requester::GET, Requester::REQUEST_MESSAGE_LIST, jsonData);
 }
 
+void Messenger::requestUserList()
+{
+    QVariantMap jsonData;
+    m_requester->sendRequest(Requester::GET,Requester::REQUEST_USER_LIST,jsonData);
```

Comment:
```suggestion
    m_requester->sendRequest(Requester::GET, Requester::REQUEST_USER_LIST, QVariantMap());
```
Code-style! Обрати внимание, слишком часто повторяется одна и та же ошибка по стилю кода